src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/utgarde_pinnacle.h

```diff
@@ -1,7 +1,14 @@
+/*
+ * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>, released under GNU AGPL v3 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-AGPL3
+ */
+
 #ifndef DEF_PINNACLE_H
 #define DEF_PINNACLE_H
 
 #include "Opcodes.h"
+#include "CreatureAIImpl.h"
+
+#define UtgardePinnacleScriptName "instance_utgarde_pinnacle"
```

Comment:
It's the same here

Reply:
https://github.com/Kitzunu/azerothcore-wotlk/blob/058aea0652d40241704db91c5a177f63f4b59d32/src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/instance_utgarde_pinnacle.cpp#L12

Reply:
```c++
#define UtgardePinnacleScriptName "instance_pinnacle"
```